wp-content/plugins/core/src/Integrations/Google_Tag_Manager/Google_Tag_Manager_Subscriber.php

```diff
@@ -10,7 +10,7 @@ public function register(): void {
 		add_action( 'wp_head', function () {
 			$this->container->get( GTM_Scripts::class )->inject_google_tag_manager_head_tag();
 		} );
-		add_action( 'tribe/body_opening_tag', function () {
+		add_action( 'wp_body_open', function () {
```

Comment:
🥇 This will make other 3rd party plugins happy.

Reply:
Tip 'o the cap to Nick ❤️